project/plugins.sbt

```diff
@@ -8,4 +8,4 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.3")
 
 addSbtPlugin("com.localytics" % "sbt-dynamodb" % "2.0.2")
 
-libraryDependencies += "org.vafer" % "jdeb" % "1.5" artifacts (Artifact("jdeb", "jar", "jar"))
+libraryDependencies += "org.vafer" % "jdeb" % "1.10" artifacts (Artifact("jdeb", "jar", "jar"))
```

Comment:
older versions of jdeb are now incompatible because of an issue with config paths in the package